api/unstable/doc.go

```diff
@@ -0,0 +1,20 @@
+/*
+Copyright 2022 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// sub-packages of api/unstable are safe to use, but are present in the 'api' package solely
+// for dependency reasons and may be changed/removed without warning. importing from any subpackage
+// of unstable may result in unexpected errors when upgrading.
+package unstable
```

Comment:
This is meant to convey "unstable" in the same sense that everything in `lib` is unstable... i.e. that we don't concern ourselves with breaking user-written code that imports it.  I do see that a lot of folks will read it as "awaiting stabilization" though.

I went with the name `unstable` because I wanted to err on the side of scary-sounding naming so that folks would be less likely to import from it and then be blindsided by changes.  @espadolini suggested the name `internal`.  Think I'll switch to that so that folks are less likely to misinterpret this as being "awaiting stabilization".

Reply:
Alright, renamed to `internal` and updated the docs a bit.  Think it should be clearer now.